content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,155 @@
+
+Both services aim for better workload protection and security guarantees by enabling memory encryption for container workloads. However, they don't seek to fully isolate the cluster or workloads against the service provider or infrastructure. Specifically, they don't offer a dedicated confidential control plane or expose attestation capabilities for the confidential cluster/nodes.
+
+Azure also enables [Confidential Containers](https://github.com/confidential-containers) in their managed Kubernetes offering. They support the creation based on [Intel SGX enclaves](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers-enclaves) and [AMD SEV-based VMs](https://techcommunity.microsoft.com/t5/azure-confidential-computing/microsoft-introduces-preview-of-confidential-containers-on-azure/ba-p/3410394).
+
+## Constellation
+
+[Constellation is a Kubernetes engine](https://github.com/edgelesssys/constellation) that aims to provide the best possible data security. It wraps your entire Kubernetes cluster into a single confidential context that is shielded from the underlying cloud infrastructure. Everything inside is always encrypted, including at runtime in memory. It shields both the worker and control plane nodes. In addition, it already integrates with popular CNCF software such as Cilium for secure networking and provides extended CSI drivers to write data securely.
+
+> TODO: Please add any interesting open-source projects in the space we should highlight!
+
+# Where are we today? Vendors, Limitations, and FOSS landscape
+
+As we have seen in the previous sections confidential computing is a powerful new concept to improve security, but we are still in the (early) adoption phase. New products are starting to emerge to take advantage of the new properties.
+
+Google and Microsoft are the first major cloud providers to have confidential offerings. Still, these offerings are limited to compute and end-to-end solutions for confidential databases, cluster networking, and load balancers have to be self-managed.
+
+At the same time using these new technologies provides us with new opportunities to bring even the most sensitive workloads into the cloud. This enables them to leverage all the tools in the CNCF landscape.
+
+# Call to action
+
+If you are currently working on a high-security product that struggles to run in the public cloud due to legal requirements or are looking to bring the privacy and security of your cloud-native project to the next level: Reach out to all the great projects we have highlighted! Everyone is keen to improve security of our ecosystem and you can play a vital role in that journey.
+
+* [Confidential Containers](https://github.com/confidential-containers)
+* [Constellation: Always encrypted K8s](https://github.com/edgelesssys/constellation)
+
+> TODO: Do we have a full list of CNCF-related CC projects? We should link to their GitHub to encourage adoption and contributions.
```

Comment:
Calling out a **TODO** here.

Reply:
maybe link to: https://confidentialcomputing.io/projects/